src/content/wiki/addons.mdx

```diff
@@ -92,6 +92,12 @@ The addon manager is a [webview](https://code.visualstudio.com/api/extension-gui
 
 If you have feedback on the addon manager, please leave it in the [vscode-lua repository](https://github.com/LuaLS/vscode-lua) <Icon name="heart" group="solid" color="red"/>.
 
+To publish your own addon to addon manager,you can refer this [Github Disscussion #2408](https://github.com/LuaLS/lua-language-server/discussions/2408)
```

Comment:
The linked discussion doesn't really have anything to do with publishing an addon. I think this part can be removed

```suggestion
```